src/Symfony/Component/Yaml/Command/LintCommand.php

```diff
@@ -102,10 +102,18 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
     private function validate($content, $file = null)
     {
+        set_error_handler(function ($level, $message) {
+            if (E_USER_DEPRECATED === $level) {
+                throw new ParseException($message);
+            }
```

Comment:
any other errors should be forwarded to the normal error handler instead of being ignored